experimental/dds/tree2/src/domains/schemaBuilder.ts

```diff
@@ -21,26 +21,25 @@ import {
 	ImplicitFieldSchema,
 	Required,
 	addFactory,
-	StructSchema,
+	ObjectNodeSchema,
 	FactoryTreeSchema,
 } from "../feature-libraries";
 import { RestrictiveReadonlyRecord, getOrCreate, isAny, requireFalse } from "../util";
 
 /**
- * A struct tree schema that satisfies the {@link SharedTreeObjectFactory} and therefore can create {@link SharedTreeObject}s.
+ * A {@link ObjectNodeSchema} that satisfies the {@link SharedTreeObjectFactory} and therefore can create {@link SharedTreeObject}s.
  * @privateRemarks
- * This type exists because TS is not able to correlate the two places where it is used if the body of this type is inlined.
+ * This type exists because TypeScript is not able to correlate the two places where it is used if the body of this type is inlined.
  * @alpha
  */
-export type FactoryStructSchema<
+export type FactoryObjectNodeSchema<
 	TScope extends string,
-	TName extends string | number,
```

Comment:
When updating this code, I noticed this parameter is needless, so I removed it.

Reply:
Thanks. I wish we could remove this whole type. It does nothing other than give a hint to TS (and bloat our exports and docs).